src/commands/_common.ts

```diff
@@ -63,14 +63,18 @@ interface Options {
  */
 export async function getSettings(options: Options = {}): Promise<Settings> {
   const { configFile } = options;
-  const tryRequire = (path: string): Settings => {
+  const tryRequire = async (path: string): Promise<Settings> => {
     // If the file is e.g. `foo.js` then Node `require('foo.js')` would look in
     // `node_modules`; we don't want this - instead force it to be a relative
     // path.
     const relativePath = resolve(process.cwd(), path);
 
     try {
-      return require(relativePath);
+      try {
+        return (await import(relativePath)).default;
```

Comment:
This fails on Node 8 but works on Node 10; and we support a minimum of Node 12 so this should be fine.

Tested via:

```
$ nvm run 8                                                                                                                                                                                 
Running node v8.17.0 (npm v6.13.4)                                                                                                                                                                                                   
> f = null; (async () => {try { f = (await import('fs')).default } catch (e) { f = require('fs') }})();                                                                                                                              
f = null; (async () => {try { f = (await import('fs')).default } catch (e) { f = require('fs') }})();                                                                                                                                
                                         ^^^^^^                                                                                                                                                                                      
                                                                                                                                                                                                                                     
SyntaxError: Unexpected token import                                                                                                                                                                                                 
                                                                                                                                                                                                                                     
>
```

```
$ nvm run 10                                                                                                                                                                                
Running node v10.20.1 (npm v6.14.4)                                                                                                                                                                                                  
> f = null; (async () => {try { f = (await import('fs')).default } catch (e) { f = require('fs') }})();                                                                                                                              
Promise {                                                                                                                                                                                                                            
  <pending>,                                                                                                                                                                                                                         
  domain:                                                                                                                                                                                                                            
   Domain {                                                                                                                                                                                                                          
     domain: null,                                                                                                                                                                                                                   
     _events:                                                                                                                                                                                                                        
      [Object: null prototype] {                                                                                                                                                                                                     
        removeListener: [Function: updateExceptionCapture],                                                                                                                                                                          
        newListener: [Function: updateExceptionCapture],                                                                                                                                                                             
        error: [Function: debugDomainError] },                                                                                                                                                                                       
     _eventsCount: 3,                                                                                                                                                                                                                
     _maxListeners: undefined,                                                                                                                                                                                                       
     members: [],                                                                                                                                                                                                                    
     [Symbol(kWeak)]: WeakReference {} } }                                                                                                                                                                                           
> f                                                                                                                                                                                                                                  
{ appendFile: [Function: appendFile],                                                                                                                                                                                                
  appendFileSync: [Function: appendFileSync],
```

```
$ nvm run 10                                                                                                                                                                                
Running node v10.20.1 (npm v6.14.4)                                                                                                                                                                                                  
> import('fs')
Promise {
  <rejected> Error: Not supported
      at repl:1:1
      at Script.runInThisContext (vm.js:122:20)
```